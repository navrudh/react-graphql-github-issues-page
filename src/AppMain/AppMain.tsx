import * as React from 'react';

import IssuesPage from '../IssuesPage/IssuesPage';

class AppMain extends React.Component {
  public render() {
    return (
      <IssuesPage />
    );
  }
}

export default AppMain;
