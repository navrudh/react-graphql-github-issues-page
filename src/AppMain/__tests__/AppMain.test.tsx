import * as React from 'react';

import AppMain from '../AppMain';
import { shallow } from 'enzyme';

describe('AppMain', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<AppMain />);
    expect(wrapper).toMatchSnapshot();
  });
});
