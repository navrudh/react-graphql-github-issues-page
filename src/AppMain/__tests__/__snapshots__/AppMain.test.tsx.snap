// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppMain renders correctly 1`] = `<IssuesPage />`;
